docs/configuration.md

```diff
@@ -303,6 +303,77 @@ profiles:
        - '1.3.6.1.2.1.1.9.1.4.*'
 ```
 
+#### 4. Configure poller to return query with additional fields present
+
+User can provide **enricher** section to make poller enrich queries sent to Splunk by adding additional dimensions. There are two types of fields:
+1. **existingVarBinds**: this section updates query results with new fields calculated from the existing SNMP information.
+2. **additionalVarBinds**: this section updates query results with additional parameters defined below.
+
+##### Existing VarBinds
+This section works only for IF-MIB oid family.
+Every property of IF-MIB family can be extracted and added as an additional dimension to the query. For example, if we want to see the name and the index of the interface along with the basic query information,
+the **enricher** must be structured as following:
+```yaml
+enricher:
+  oidFamily:
+    IF-MIB:
+      existingVarBinds:
+        - ifIndex: 'interface_index'
+        - ifDescr: 'interface_desc'
+```
+From now on in every query for IF-MIB two new fields "interface_index" and "interface_desc" are visible:
+```yaml
+oid-type1="ObjectIdentity" value1-type="Integer" 1.3.6.1.2.1.2.2.1.7.2="1" value1="1" IF-MIB::ifAdminStatus.2="up" interface_index="2" interface_desc="eth0"
+```
+The value of newly added properties is calculated according to current query index, for IF-MIB::ifAdminStatus.**2** we're interested in IF-MIB::ifIndex.**2** and IF-MIB::ifDescr.**2**.
```

Comment:
I think it would be easier to understadn divided into 2 sentences 

The value of newly added properties is calculated according to current query index. 
For example, for IF-MIB::ifAdminStatus.**2** we're interested in IF-MIB::ifIndex.**2** and IF-MIB::ifDescr.**2**.